tests/bson/bson-timestamp_error-003.phpt

```diff
@@ -10,15 +10,18 @@ echo throws(function() {
 }, 'MongoDB\Driver\Exception\InvalidArgumentException'), "\n";
 
 echo throws(function() {
-    new MongoDB\BSON\Timestamp(-2147483648, 0);
+    /* I realise that "-2147483647 - 1" could be written as "-2147483648", *however*, PHP considers
+     * the latter a floating point number, as it parses "-" and "2147483648" separately, and
+     * "2147483648" doesn't fit in the 32-bit signed range. */
+    new MongoDB\BSON\Timestamp(-2147483647 - 1, 0);
```

Comment:
I this still going to produce an exception with "Expected increment to be an unsigned 32-bit integer, -2147483648 given" on 32-bit systems?

Reply:
Yes. That's exactly what this test tests :-)